### ISSUE TITLE
[BUG] - The "PWD" variable is not set. Defaulting to a blank string. 

### ISSUE THREAD

**Issue by leonardocintra** [rodrigovp/livro-kafka-connect#1]
https://github.com/rodrigovp/livro-kafka-connect/blob/9ee5fdb0284420df5397d62771945f776565be08/docker/docker-compose.yml#L147

**Comment by deywillan** [comment]
@leonardocintra Conseguiu resolver? Ainda estou nesse problema.

**Comment by deywillan** [comment]
Eu resolvi o problema alterando a imagem cdc-kafka-connect para confluentinc/cp-kafka-connect

**Comment by leonardocintra** [comment]
@deywillan sim. Eu na verdade estou sempre usando as ultimas versões que ele usou no livro, que pra mim ja estao mais antigas... mas ainda da pau umas coisas... mas precisava tirar uma duvida em algum lugar, ate mandei email pra ele e fui respondido, mas era bom as vezes termos uma comunidade para nos ajudarmos
